autotest.pm

```diff
@@ -317,7 +317,7 @@ sub query_isotovideo {
     }
     $json{cmd} = $cmd;
 
-    # send the command to isotovideo
+    die "isotovideo is not initialized. Ensure to call test API calls only from test modules, not schedule code\n" unless defined $isotovideo;
```

Comment:
```suggestion
    die "isotovideo is not initialized. Be sure to only call test API from test modules, not schedule code\n" unless defined $isotovideo;
```

Reply:
That's wrong. Now we are missing an article and I think you misjudged "calls" as being a verb but I meant it as plural of "test API calls" :)

Reply:
I don't see any missing articles, I just changed it because "ensure to call" feels wrong to me.

How about this:

    ```suggestion
    die "isotovideo is not initialized. Please ensure that you only call test API from test modules, not schedule code\n" unless defined $isotovideo;
```

Reply:
That's still missing an article and not proper english. "ensure that you only call test API" should rather say "ensure that you only call the test API". But again the word "calls" is missing. We could replace "calls" with "functions" though. So

```suggestion
    die "isotovideo is not initialized. Ensure that you only call test API functions from test modules, not schedule code\n" unless defined $isotovideo;
```

Reply:
Thank you for saying what you meant. Sounds good to me

Reply:
done